src/main/java/hw7/pages/JDIMetalsAndColors.java

```diff
@@ -0,0 +1,157 @@
+package hw7.pages;
+
+import com.epam.jdi.light.elements.complex.Checklist;
+import com.epam.jdi.light.elements.complex.WebList;
+import com.epam.jdi.light.elements.complex.dropdown.Dropdown;
+import com.epam.jdi.light.elements.composite.WebPage;
+import com.epam.jdi.light.elements.pageobjects.annotations.FindBy;
+import com.epam.jdi.light.elements.pageobjects.annotations.Title;
+import com.epam.jdi.light.elements.pageobjects.annotations.Url;
+import com.epam.jdi.light.elements.pageobjects.annotations.locators.JDropdown;
+import com.epam.jdi.light.ui.html.elements.common.Button;
+import hw7.entities.MetalsAndColorsData;
+import hw7.forms.MetalsAndColorsForm;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+
+import java.util.Arrays;
+import java.util.List;
+
+
+@Url("metals-colors.html") @Title("Metal and Colors")
+public class JDIMetalsAndColors extends WebPage {
```

Comment:
I don't get it. You already have a MetalsAndColorsForm, what is the reason of similar elements in the Page ??

Reply:
MetalsAndColorsForm needs too many time for work. I don`t know reason of this situation, but first test case have continued for 5-6 minutes.
MetalsAndColorsForm class is here to show, that i tried to use this way. It can be deleted.